src/device-registry/controllers/create-activity.js

```diff
@@ -70,6 +70,7 @@ const activity = {
           message: result.message,
           createdActivity: result.data.createdActivity,
           updatedDevice: result.data.updatedDevice,
+          network: result.data.network,
```

Comment:
Hello @AnthonyByansi :

Thank you for your contribution. 

However, I foresee potential runtime issues (errors) that could disrupt the service due to the absence of network-related functionality in the utilised method, as evidenced by the attached screenshot. 

It's crucial to manually test functionalities, especially for critical services like device-registry and auth-service, before submitting PRs.

cc: @BenjaminSsempala  
<img width="1277" alt="Screenshot 2024-07-22 at 11 00 36" src="https://github.com/user-attachments/assets/e2f48675-91ce-4dab-a237-1457cae0419c">

Reply:
Thanks @Baalmart for your constructive feedback!
I will definitely carry out the manual tests and submit my findings afterwards.

Cheers 🥂